### ISSUE TITLE
[ Navbar + Pill + Tabs ] Navbar with Pills or Tabs

### ISSUE THREAD

**Issue by 0wczar** [twbs/bootstrap#21520]
This is what I posted below is very often seen in the case of UI Dashboards.

So my question:
- Can we add "Pills" in the navbar?
- ~~Can we add „Tabs" to navbar?~~ _- found in „Scrollspy” section_ ( http://v4-alpha.getbootstrap.com/components/scrollspy/ )

What do you think?
![zrzut ekranu 2017-01-03 17 13 57](https://cloud.githubusercontent.com/assets/2330394/21614105/71e312a6-d1d8-11e6-9bd3-52577d84a385.png)

**Comment by alextarkhov** [comment]
Hello @0wczar I can add this component, if the community does not implement in v4. I'm working on an alternative library [PostBoot](https://alextarkhov.github.io/postboot/) that contains components that are not included in original Bootstrap sources or removed earlier for some reasons.

**Comment by Olivia101** [comment]
@alextarkhov It would have been awesome if the JavaScript in PostBoot was vanilla JS and not jQuery.

**Comment by alextarkhov** [comment]
Thank you @Olivia101 I rewrite components to native JS in next alpha release. In issue #21416 I proposed to remove jQuery from v4.

**Comment by 0wczar** [comment]
@alextarkhov If I find the time - I'm going to implement to v4.

**Comment by alextarkhov** [comment]
@0wczar It's be a cool.

**Comment by 0wczar** [comment]
@bardiharborow 
I found **Navbar with Pills**. Located in the section **"Scrollspy”** ( http://v4-alpha.getbootstrap.com/components/scrollspy/ ) -

I think that this example **”Navbar with Pills”** should also be in the section **"Navbar”**?

**Comment by a207052002** [comment]
Sorry, I have another issue about pill.(v4 alpha6)
And if it needs, I will put the demo out.
But here I tried to ask first.
First, the 'focus' or 'active' in nav-pills don't work well. It often loses its blue while clicking.
And I tried to override the class('active', 'focus', 'hover' ) , and I found out that the pills only reacted correctly about hover...
I guessed the codes of v4 are totally different from v3, so I couldn't easily change the color through CSS.
I 'm such a noob that I can't dig out more.
